pkg/docker/save.go

```diff
@@ -30,7 +30,15 @@ func Save(image string, dir string) error {
 	if err != nil {
 		return (err)
 	}
-	events, err := cli.ImagePull(ctx, image, types.ImagePullOptions{})
+	var ipo types.ImagePullOptions
+	// Load auth details from .docker config
+	authStr := GetAuth(image)
+	if len(authStr) > 0 {
+		ipo = types.ImagePullOptions{RegistryAuth: authStr}
+	} else {
+		ipo = types.ImagePullOptions{}
```

Comment:
i think this is doing the same thing as `var ipo types.ImagePullOptions` ... perhaps we can shrink the lines to 
```Go
var ipo types.ImagePullOptions
if len(authStr) > 0 { 
   ipo.RegistryAuth = GetAuth(image)
}
```

Reply:
Yep - works - declare = nil things, lib is happy.